cdap-docs/_common/common-build.sh

```diff
@@ -279,12 +272,21 @@ function check_includes() {
 }
 
 function download_includes() {
+  local includes_dir=${1}
```

Comment:
Because this function is a template to be over-shadowed in the inner manuals that use this. I was thinking I could then drop the comment, as this documents what $1 is.